packages/react-core/src/demos/RTL/examples/PaginatedTable.tsx

```diff
@@ -133,17 +141,17 @@ export const PaginatedTableAction = () => {
 
   // Pagination logic
 
-  const handleSetPage = (_evt, newPage, _perPage, startIdx, endIdx) => {
+  const handleSetPage = (_evt: Event, newPage: number, _perPage: number, startIdx: number, endIdx: number) => {
     setPaginatedRows(managedRows.slice(startIdx, endIdx));
     setPage(newPage);
   };
 
-  const handlePerPageSelect = (_evt, newPerPage, _newPage, startIdx, endIdx) => {
+  const handlePerPageSelect = (_evt: Event, newPerPage: number, _newPage: number, startIdx: number, endIdx: number) => {
```

Comment:
These types don't quite match what is expected for the Pagination props they're being passed to. You can actually import the types for these callbacks directly to avoid having to manually type them here.

Reply:
Yes, some params are optional in callback. I'll fix it.